frontend/src/scenes/persons/PersonDeleteModal.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { personsLogic } from './personsLogic'
+import { asDisplay } from './PersonHeader'
+import { LemonButton, LemonModal } from '@posthog/lemon-ui'
+import { PersonType } from '~/types'
+
+export function PersonDeleteModal(): JSX.Element | null {
+    const { personDeleteModal } = useValues(personsLogic)
+    const { deletePerson, showPersonDeleteModal } = useActions(personsLogic)
+
+    return (
+        <LemonModal
+            isOpen={!!personDeleteModal}
+            onClose={() => showPersonDeleteModal(null)}
+            title={`Are you sure you want to delete "${asDisplay(personDeleteModal)}"?`}
+            description={
+                <>
+                    This action cannot be undone. If you opt to delete the person and its corresponding events, the
```

Comment:
I'd add a line break after "undone". Seems like the next sentence is about to offer a further explanation about the first one